app/github_public_repo/README.md

```diff
@@ -0,0 +1,20 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+SeroTracker synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset represents our collection of serosurveillance studies
+
+### Download
+Our complete SARS-Cov-2 dataset is available as a CSV programmatically with the following command:
+- `curl https://raw.githubusercontent.com/serotracker/sars-cov-2-data/main/test.csv`.
+
+Otherwise, you can also download our dataset directly from our [Airtable view]([https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on](https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on)).
+
+The CSV file follows the format of 1 row per serosurveillance study.
+
+  
+### Additional Information
+Please see our [Data Dictionary]([https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0](https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0)) for explanations of our variables, data types, and descriptions as well as insight into how our data is collected by our research team.
+
+In order to keep up to date with important changes to our dataset, please consult our [Change Log]([https://airtable.com/shrxpAlF6v0LeRYkA/tblC6jj904WXUzwVY](https://airtable.com/shrxpAlF6v0LeRYkA/tblC6jj904WXUzwVY)) regularly.
```

Comment:
You copied the URL twice here

Reply:
Are we sure that this change log will capture all of the changes relevant to the CSV? i.e. there isn't a case in which there will be a change made to the csv that won't be represented in the change log? Just want to make sure we have confirmed this with the research team.

Reply:
yup! confirmed with Christian who will be one of the main maintainers

Reply:
Updated: the readme changelog will now be maintained by gabe on the research team who will also be maintaining the corresponding airtable table containing the field names.